tests/test-cases/Group6-VIC-Machine/6-03-Delete.robot

```diff
@@ -43,6 +42,23 @@ Cleanup Delete Tests
     Run Keyword If  '${tempvm}'!='${EMPTY}'  Log  ${output}
     Run Keyword If  '${tempvm}'!='${EMPTY}'  Should Be Equal As Integers  ${rc}  0
 
+Cleanup Delete VCH with non-cVM in VCH folder test
+    Run Keyword And Ignore Error  Cleanup VIC Appliance On Test Server
+
+    Set Environment Variable  TEST_DATACENTER  ${orig-dc}
```

Comment:
not clear what you are doing here and why?

Reply:
@anchal-agrawal Having looked at the test in question, it doesn't appear as though other tests are depending on the modified value, so why not create a test variable that holds the reified datacenter value if necessary.
Also - if there's only one datacenter (which is the only valid case for TEST_DATACENTER being nil) then I think you can use `*` but that should be confirmed.